app/uk/gov/hmrc/fileupload/s3/S3JavaSdkService.scala

```diff
@@ -61,6 +61,9 @@ class S3JavaSdkService(configuration: com.typesafe.config.Config, metrics: Metri
     .withProxyPort(awsConfig.proxyPort)
     .withProxyUsername(awsConfig.proxyUsername)
     .withProxyPassword(awsConfig.proxyPassword)
+    .withConnectionTimeout(5 * 1000) // FIXME pull to config
```

Comment:
did you mean to commit the FIXME?

Reply:
firstly I want to test if this would fix the QA.
If yes then I'll pull those magic numbers to config